graphql/README.md

```diff
@@ -41,24 +28,22 @@ be used.
   Tools](https://www.apollographql.com/docs/react/features/developer-tooling)**
   Chrome Extension offering developer tools for Apollo projects.
 
-## Vocabulary
-
-## Patterns
-
-### Apollo
-
-### Relay
-
 ## Best Practices
 
-## Recommendations
-
-### Error Reporting
-
-### Deleting Resources
-
-## Authn, Authz
-
-## Open Questions
-
-
+* Follow the latest version of the [GraphQL specification].
+* When serving over HTTP, respond with a 200 OK status code to all GraphQL
```

Comment:
We're blocking unauthenticated requests to the GraphQL API on ADS with a 401. Is that a valid case for a non-200 response?

Reply:
[No](https://github.com/rmosolgo/graphql-ruby/issues/1130#issuecomment-347373937):

> Non-200 response codes indicate that some issue occurred at the HTTP transport layer

All GraphQL responses that we are in control of should be 200s.

Reply:
Yes, after more experience with GraphQL, I agree that we didn't do it the best way on that project.